src/store/reducers/members.ts

```diff
@@ -305,14 +312,22 @@ function addMembersToState(
  * TODO (code cleanup): Avoid having to assert on id present for almost every operation type.
  */
 function applyOperation(
-  prevState: MembersState,
```

Comment:
I'd rather you just left this as `prevState` to not reassign parameters, and then later on just wrote

```
const newState = creator ?
  addMemberToState(prevState, creator.updateLastOpCreatedAt(createdAt)) : 
  prevState
```

and refer to that. no need to compromise here, gains nothing and muddies the meaning of the parameter (is it the original state or some working modified copy of it?).